app/src/main/java/com/framgia/f_talk/data/model/User.java

```diff
@@ -0,0 +1,131 @@
+package com.framgia.f_talk.data.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class User {
+    private String mUId;
+    private String mFullName;
+    private String mEmail;
+    private String mAvatarUrl;
+    private String mBio;
+    private String mLocation;
+    private String mDateOfBirth;
+    private List<String> mListGroupIds;
+    private List<String> mListFriendUIds;
+    private List<String> mListIncomeRequests;
+    private List<String> mListOutComeRequest;
+
+    public User() {
+
+    }
+
+    public User(String UId, String fullName, String email, String avatarUrl) {
+        mUId = UId;
+        mFullName = fullName;
+        mEmail = email;
+        mAvatarUrl = avatarUrl;
+        mListGroupIds = new ArrayList<>();
```

Comment:
```suggestion
        mGroupIds = new ArrayList<>();
```
Em không nên khởi tạo ở đây. khi nào dùng thì khởi tạo. tương tự vơi các trường khác